editor.planx.uk/src/pages/FlowEditor/components/Team/components/EditorUpsertModal.tsx

```diff
@@ -58,12 +56,15 @@ export const EditorUpsertModal = ({
   const handleSubmitToAddNewUser = async () => {
     const { teamId, teamSlug } = useStore.getState();
 
+    if (formik.values.email === null) {
+      return;
+    }
```

Comment:
This looks like a type guard, which I think we can solve via a more semantically meaningful method (see previous comment).